watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -206,10 +114,10 @@ def test_costing(self, system_frame):
         m = system_frame
 
         # check costing
-        assert value(m.fs.costing.LCOW) == pytest.approx(0.46904, rel=1e-3)
+        assert value(m.fs.costing.LCOW) == pytest.approx(0.46814, rel=1e-3)
         assert value(m.fs.costing.total_capital_cost) == pytest.approx(
-            23985398.4909, rel=1e-3
+            23939185.51, rel=1e-3
```

Comment:
I think it would be better to use a smaller tolerance for this large value. A tolerance of 0.1% can still represent a significant absolute difference.